yarp.gemspec

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+Gem::Specification.new do |spec|
+  spec.name = "yarp"
+  spec.version = "0.4.0"
+  spec.authors = ["Shopify"]
+  spec.email = ["ruby@shopify.com"]
+
+  spec.summary = "Yet Another Ruby Parser"
+  spec.homepage = "https://github.com/ruby/yarp"
+  spec.license = "MIT"
+
+  spec.require_paths = ["lib"]
+  spec.files = [
+    "CODE_OF_CONDUCT.md",
+    "CONTRIBUTING.md",
+    "LICENSE.md",
+    "Makefile.in",
+    "README.md",
+    "config.h.in",
+    "config.yml",
+    "configure",
+    "ext/yarp/api_node.c",
+    "include/yarp/ast.h",
+    "java/org/yarp/AbstractNodeVisitor.java",
+    "java/org/yarp/Loader.java",
+    "java/org/yarp/Nodes.java",
```

Comment:
We don't need those in the gem, at least not for TruffleRuby.

Reply:
@enebo Do these need to be in the gem for JRuby?

Reply:
I would say not for now for JRuby, because there is nothing compiling these `.java` files in the gem right now.
We can always include them later if it's useful.

Reply:
OK, will remove.